### PR TITLE
Test tabulate 0.7.6dev

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

File: tests/test_io.py [hunk 1]
```diff
@@ -348,6 +343,28 @@ class TestRst(TestIO):
             =====  =  =  =""").strip('\n')
     }
 
+    if (LooseVersion(tabulate.__version__) <= LooseVersion('0.7.5') or
+            platform.python_implementation() == 'PyPy'):
```
Comment (jayvdb) [inline tests/test_io.py:347]: this pypy specific difference is really weird ... need to raise that bug upstream

File: tests/test_io.py [hunk 2]
```diff
@@ -451,6 +468,12 @@ class TestHTML(TestIO):
             """).strip('\n'),
     }
 
+    # tabulate 0.7.6-dev adds <thead> and <tbody> elements
+    if LooseVersion(tabulate.__version__) > LooseVersion('0.7.5'):
+        expected_results = dict(
+            (key, value.replace('<tr><th ', '<thead>\n<tr><th ').replace('<tr><th>', '<thead>\n<tr><th>').replace('</th></tr>', '</th></tr>\n</thead>\n<tbody>').replace('<table>\n<tr>', '<table>\n<tbody>\n<tr>').replace('</table>', '</tbody>\n</table>'))
```
Comment (jayvdb) [inline tests/test_io.py:474]: this is very ugly.  need to think of a better way to do this.

maybe just have this test class raise an error for 0.7.6+ , except on Travis where it skips the test class

**Comment by chfw** [comment]
do we really want test pyexcel-text againt both 0.7.5 and 0.7.6?

**Comment by jayvdb** [comment]
When I am debugging tabulate issues, i am usually usually switching between tabulate 0.7.5 and master, and it is annoying to see many errors which are known improvements of 0.7.6 .  When 0.7.6 is released, I expect we would want to support 0.7.6 and 0.7.5 , especially as 0.7.5 is what is packaged by Ubuntu and Fedora (and we should also test 0.7.4 too, but that is less important.)  So if we need to change our tests when 0.7.6 is released, why not change them now.

**Pull request merged**